test/misc.jl

```diff
@@ -15,6 +15,13 @@ vals, lens = rle(z)
 @test lens == [2, 2, 1, 1, 3]
 @test inverse_rle(vals, lens) == z
 
+# levelsmap
+y = [1, 1, 2, 2, 2, 3, 1, 2, 2, 3, 3, 3, 3]
+
+@test levelsmap(y) == Dict([(2, 2), (3, 3), (1, 1)])
+@test levelsmap(z) == Dict([(false, 2), (true, 1) ])
```

Comment:
Probably better use a new vector here instead of `z`, which is redefined in each block above. It would be clearer (you can even use a `let` block around the two instructions you added).